blocks/templates/blocks.pot

```diff
@@ -1,495 +1,613 @@
+#. #-#-#-#-#  commands-online.pot (PACKAGE VERSION)  #-#-#-#-#
+#. extracted from commands-online.txt
+#. #-#-#-#-#  commands-offline.pot (PACKAGE VERSION)  #-#-#-#-#
+#. extracted from commands-offline.txt
+#, fuzzy
 msgid ""
 msgstr ""
 "Project-Id-Version: PACKAGE VERSION\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2014-04-25 22:29+0200\n"
-"PO-Revision-Date: 2013-05-02 00:42+0000\n"
-"Last-Translator: adlogi <adlogi@media.mit.edu>\n"
+"POT-Creation-Date: 2015-04-13 13:04-0700\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
 "Language-Team: LANGUAGE <LL@li.org>\n"
-"Language: templates\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
-"X-Generator: Pootle 2.5.1.1\n"
-"X-POOTLE-MTIME: 1367455321.000000\n"
-
-# The language name as it will appear in the language menu
-msgid "Language-Name"
-msgstr ""
+"X-Generator: Translate Toolkit 1.12.0\n"
 
+#: commands-online.txt:1 commands-offline.txt:1
 msgid "%b and %b"
 msgstr ""
 
+#: commands-online.txt:3 commands-offline.txt:3
 msgid "%b or %b"
 msgstr ""
 
+#: commands-online.txt:5 commands-offline.txt:5
 msgid "%m.attribute of %m.spriteOrStage"
 msgstr ""
 
-msgid "%m.list contains %s"
+#: commands-online.txt:7 commands-offline.txt:7
+msgid "%m.list contains %s?"
 msgstr ""
 
+#: commands-online.txt:9 commands-offline.txt:9
 msgid "%m.mathOp of %n"
 msgstr ""
 
+#: commands-online.txt:11 commands-offline.txt:11
 msgid "%m.sensor sensor value"
 msgstr ""
 
+#: commands-online.txt:13 commands-offline.txt:13
 msgid "%n mod %n"
 msgstr ""
 
+#: commands-online.txt:15 commands-offline.txt:15
 msgid "add %s to %m.list"
 msgstr ""
 
+#: commands-online.txt:17 commands-offline.txt:17
 msgid "answer"
 msgstr ""
 
+#: commands-online.txt:19 commands-offline.txt:19
 msgid "ask %s and wait"
 msgstr ""
 
-# "backdrop" in Scratch 2.0 is similar to "background" in Scratch 1.4
```

Comment:
It's a translator-comment - it may have been put in manually. Probably it's not relevant now (this particular one). However, later it may be good to have a way for pulling in translator-comments in the editor code.